config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -178,6 +178,17 @@ spec:
                 deprecated: as of 4.7, this field is no longer set or honored.  It
                 will be removed in a future release.'
               type: string
+            highAvailabilityMode:
+              description: HighAvailabilityMode express the high-availability expectations.
+                The default is 'Full', which represents the behavior operators have
+                in a \"normal\" cluster. The 'None' mode will be used in single-node
+                deployments (developer and production) for example, and the operators
+                should not configure the operand for highly-available operation
+              type: string
+              enum:
+              - ""
+              - Full
```

Comment:
how about something along ThreeNode ?

Reply:
I think in the enhancement we want to have either full or non-ha. How ThreeNodes will help?

Reply:
is there half HA ? APIs should be clear when you read them. Full and None have no meaning.

Reply:
https://github.com/openshift/enhancements/pull/555/files#diff-47d36bb8151b04437b398146ca27efc44639a74bb22faef61f0c54bc6f193505R82

Reply:
Thanks for the link. Let's move the bigshedding there. I should have guessed that this field has been bikeshedded to death already ;-)